server/webhook.go

```diff
@@ -19,6 +20,12 @@ const (
 )
 
 func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+
+	if p.configuration.WebhookSecret != params["webhooksecret"] {
```

Comment:
Should this use `crypto/subtle.ConstantTimeCompare`?

Reply:
both were string so directly compared

Reply:
Well, converting a string to a byte slice is not a big deal, right?

Reply:
Sure. have done this